### ISSUE TITLE
Release status

### ISSUE THREAD

**Issue by guiccbr** [drcallaway/apollo-link-timeout#38]
Hi, hello. Thanks for the amazing job here.

The master branch has a few fixes that allows this lib to be used with Apollo 3, but the latest release seems to be from 2019. It would be nice if we could generate a new release from master to allow folks to use the latest version in their projects without the need of forking. Is there a plan for that?

Thank you!

**Comment by drcallaway** [comment]
Thanks @guiccbr! I'll do a release this evening.

**Comment by drcallaway** [comment]
Thanks for your help with this @guiccbr. I went ahead and released master as v4.0.0. Please give it a try and let me know if you run into any issues.

**Comment by guiccbr** [comment]
Great, @drcallaway, thanks!

**Comment by guiccbr** [comment]
@drcallaway, when you mentioned v4.0.0 I went to https://github.com/drcallaway/apollo-link-timeout/tags to see if there was a new release tag, but didn't find it there.

Then I got it: my package.json was using the latest release number that was show in the repo `^1.3.1` and I never looked for new releases on npm directly: https://www.npmjs.com/package/apollo-link-timeout/v/4.0.0. But they were all there. I could have used v3.0.0, so sorry for that.
